source/geometries/Next100.cc

```diff
@@ -38,8 +38,9 @@ namespace nexus {
     lab_size_ (5. * m),
 
     // common used variables in geomety components
-    gate_tracking_plane_distance_(30. * mm), // to be confirmed
-    gate_sapphire_wdw_distance_  (1460.5 * mm),
+    // 0.1 mm gate thickness?
```

Comment:
Do you mean grid thickness?

Reply:
Right. I have modified the comment to clarify.